.github/workflows/osx_build.yml

```diff
@@ -0,0 +1,52 @@
+name: MacOS build and test rolesanywhere-credential-helper
+on:
+  push:
+    branches: [ main ]
+  
+permissions:
+  statuses: write
+  id-token: write
+
+jobs:
+  OSX:
+    name: Build on ${{matrix.runner_image}}
+    runs-on: ${{matrix.runner_image}}
+    strategy:
+      matrix:
+        include:
+          - runner_image: macos-13
+            s3_path: X86_64/OSX/MacOS
```

Comment:
"OSX" and "MacOS" are effectively synonymous. Do we need to have both in the S3 path?